### DISCUSSION TITLE
С включенным zapret не работает Geforce Experience

### DISCUSSION THREAD

**Discussion by Namefree** [bol-van/zapret#853] [Q&A]
Какой есть способ, что бы работал Geforce Experience с моими настройками не работает выдает ошибку.
`start "zapret: discord" /min "%~dp0winws.exe" ^
--wf-tcp=80,443,50000-65535 --wf-udp=443,50000-65535 ^
--filter-udp=443 --hostlist="%~dp0list-general.txt" --dpi-desync=fake --dpi-desync-udplen-increment=10 --dpi-desync-repeats=6 --dpi-desync-udplen-pattern=0xDEADBEEF --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^
--filter-udp=50000-65535 --dpi-desync=fake,tamper --dpi-desync-any-protocol --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^
--filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^
--filter-tcp=443 --hostlist="%~dp0list-general.txt" --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^
--dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig`

**Comment by bol-van** [comment]
Очередной раз отмечаю ТРЭШ С ФЛУДОМ из-за отсутствия cutoff

**Comment by TrueKoldun** [comment]
nvidia.com в исключения добавь через --hostlist-exclude=[путь до файла с исключениями]

**Comment by MankoAlexei** [comment]
Попробовав много способов выяснил что как бы я не хотел он не заработает. winws.exe попросту спустя время выключает Geforce experience с ошибкой 0x0002.  Я добавлял сайт nvidia в исключение, работает zapret только через service_discord_youtube.bat . В службах тоже делал финты ушами.

**Comment by numbereleven-a** [comment]
cutoff + порты уменьш до 50100
